docs/examples/usage/test_configuration_12.py

```diff
@@ -0,0 +1,16 @@
+def test_basic_statement_config() -> None:
+    from sqlspec.adapters.asyncpg import AsyncpgConfig
+    from sqlspec.core.statement import StatementConfig
```

Comment:
```suggestion
    from sqlspec.core import StatementConfig
```

I think i've been able to get everything that you need publicly from the `core.**.*` at the `sqlspec.core` layer.  I'm not sure if this is a good idea, but the intent was to prevent import sprawl.